pkg/resources/resource_source_postgres.go

```diff
@@ -34,8 +35,8 @@ var sourcePostgresSchema = map[string]*schema.Schema{
 		Optional:    true,
 	},
 	"table": {
-		Description: "Creates subsources for specific tables. If neither table or schema is specified, will default to ALL TABLES",
-		Type:        schema.TypeList,
+		Description: "Creates subsources for specific tables in the Postgres connection.",
+		Type:        schema.TypeSet,
```

Comment:
Has to be `TypeSet` instead of `TypeList` since we cannot control order set by the user vs order returned by the scan query.